biomage/rds/migrate.py

```diff
@@ -0,0 +1,83 @@
+import os
+import subprocess
+from pathlib import Path
+
+import click
+
+from ..utils.AuroraClient import AuroraClient
+from ..utils.constants import DEFAULT_AWS_ACCOUNT_ID, DEVELOPMENT, STAGING
+
+# Assuming that biomage-utils and iac root folders are located in the same folder
+MODULE_PATH = os.path.dirname(os.path.abspath(__file__))
+DEFAULT_IAC_PATH = os.path.join(MODULE_PATH, "../../../iac")
+IAC_PATH = os.getenv("BIOMAGE_IAC_PATH", DEFAULT_IAC_PATH)
+
+
+def _migrate(iac_path, migration_env):
+    proc = subprocess.Popen(
+        ["node_modules/.bin/knex", "migrate:latest", "--cwd", iac_path],
+        cwd=iac_path,
+        env=migration_env,
+    )
+    proc.wait()
+
+
+@click.command()
+@click.option(
+    "-i",
+    "--input_env",
+    required=True,
+    default=DEVELOPMENT,
+    show_default=True,
+    help="Path to the IAC folder",
+)
+@click.option(
+    "-s",
+    "--sandbox_id",
+    required=False,
+    default=None,
+    show_default=True,
+    help="Sandbox id to migrate to. Required if migrating to staging",
+)
+@click.option(
+    "--iac_path",
+    required=False,
+    default=IAC_PATH,
+    show_default=True,
+    help="Path to the IAC folder",
+)
+def migrate(iac_path, sandbox_id, input_env):
+    """
+    Migrate database in environment
+    """
+
+    REGION = "eu-west-1"
+    AWS_PROFILE = "default"
+    AWS_ACCOUNT_ID = "000000000000"
+    USER = "dev_role"
+    LOCAL_PORT = 5431
+
+    if input_env == STAGING:
+        AWS_ACCOUNT_ID = DEFAULT_AWS_ACCOUNT_ID
+        LOCAL_PORT = 5432
+
```

Comment:
```suggestion
if command is None:
    command = "migrate:latest"
```

Reply:
We can't do this, as `click.argument` makes the argument a required argument, unlike `click.option`. The command will fail before reaching this step.

Reply:
We can make this an option

Reply:
Added this as an option `-c` with a default of `migrate:latest`